src/report.rs

```diff
@@ -72,6 +74,9 @@ pub struct Settings {
     /// List of advisory IDs to ignore
     pub ignore: Vec<advisory::Id>,
```

Comment:
Would suggest changing this to disambiguate:

```suggestion
    pub ignored_advisories: Vec<advisory::Id>,
```